Source/SwiftLintFramework/Rules/VerticalWhitespaceBetweenCasesRule.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+import SourceKittenFramework
+
+private extension File {
+    func violatingRanges(for pattern: String) -> [NSRange] {
+        return match(pattern: pattern, excludingSyntaxKinds: SyntaxKind.commentAndStringKinds)
+    }
+}
+
+public struct VerticalWhitespaceBetweenCasesRule {
+    public init() {}
+
+    private static let violatingToValidExamples: [String: String] = [
+    """
+        switch x {
+        case 0..<5:
+            print("x is valid")
+        default:
+            print("x is invalid")
+        }
+    """: """
+        switch x {
```

Comment:
It's missing is the location where the violations are triggered. The examples should have "↓" to indicate where the violation is triggered. The Unit Tests will use the location of the arrow to make sure the violations are triggered on the correct place.

Reply:
Done.